TelepresenceBot/mobile/src/main/java/com/novoda/tpbot/bot/BotActivity.java

```diff
@@ -78,6 +78,13 @@ public void onLazersReleased() {
         }
     };
 
+    private final ServerDeclarationListener serverDeclarationListener = new ServerDeclarationListener() {
```

Comment:
Wait a second, why do we need this here?
The bot _is_ the server or did I miss something?
Are we considering having a man-in-the-middle server?